StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -354,18 +353,28 @@ extension LinkAccountPickerViewController: LinkAccountPickerBodyViewDelegate {
         FeedbackGeneratorAdapter.selectionChanged()
 
         let selectedPartnerAccount = selectedAccountTuple.partnerAccount
+        let eligibleToPresentAccountUpdateRequiredDrawer: Bool = (
+            // repair flow
+            selectedPartnerAccount.nextPaneOnSelection == .bankAuthRepair
+                // supportability -- account requires re-sharing with additonal permissions
+                || selectedPartnerAccount.nextPaneOnSelection == .partnerAuth
+        )
```

Comment:
this logic was moved from the bottom of the file, but [stripe.js here also](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/pull/23288/files#diff-96f25fd92e7a6ede67b5de1870616635186357cb3642022af8a41e0c25a968faR217) and [here too](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/b29037a9/src/linkedAccounts/inner/components/panes/v3/LinkAccountPickerPane/lib/helpers.ts#L3-L12)